BP/history/provinces/bp_k_oriens.txt

```diff
@@ -1,22 +1,36 @@
-﻿# k_oriens
+﻿# Notes:
+# 500.1.1: Apolloniarianism died out in the 4th century
+# 518.1.1: Valentinianism dies out in Aramean provinces
+# 550.1.1: Arab Doceticism in Siani merges into Ebionites (who probably survived in NW Arabia until 1000)
+# 550.1.1: Greek Doceticism turns miaphysitism
+# 518.1.1: Minority Aramean Christians turn Miaphysite instead
+# 550.1.1: Assyrian paganism dies out and is split between nestorian (persian) and miaphysitism (roman)
+#	Palmyra remains the exception
```

Comment:
Feels like Assyrian Pagan got a lot of territories as a consequences of the increased province density while it was more contained in WtWSMS, not sure it should be that prevalent when reviewing the map.

Reply:
Addressed with 87727f9. Assyrian culture slightly shifted to better match historical center of Assyria. Aramean Patripassianism and Aramean Audianism replace some of the western assyrian_pagan provinces.